py_irt/dataset.py

```diff
@@ -40,7 +42,7 @@ def get_item_accuracies(self) -> Dict[str, ItemAccuracy]:
         return item_accuracies
 
     @classmethod
-    def from_jsonlines(cls, data_path: Path):
+    def from_jsonlines(cls, data_path: Path, train_items: dict = None):
```

Comment:
This is fine for now, but we should allow a way to specify this as a data file so we can support train in command, persisting everything to disk, and then eval in another.